pytrack/core/py_track.py

```diff
@@ -10,57 +10,53 @@
 """
 
 from __future__ import annotations
-from typing import Union, List
 
 import logging
-import json
 import subprocess
 import yaml
-from pathlib import Path
-import abc
 
-from .dataclasses import DVCParams, SlurmConfig
-from .parameter import parameter, result
-from pytrack.utils import is_jsonable
+from .data_classes import SlurmConfig
```

Comment:
Have we done any slurm testing?

Reply:
Yes, it uses `srun -n n_configs python -f from <module> import <stage>; <stage>.(id_=0).run()`